apps/api/v1/pages/api/bookings/_get.ts

```diff
@@ -186,16 +181,7 @@ function buildWhereClause(
     };
   } else {
     whereClause = {
-      OR: [
-        userFilter,
-        {
-          attendees: {
-            some: {
-              email: { in: userEmails },
-            },
-          },
-        },
-      ],
+      ...userFilter,
```

Comment:
Isn't this spread not needed and you could write :
```ts
whereClause = userFilter
```
?

Reply:
Indeed. Likely left over from a refactor. Will clean up when I pick this back up. Thanks.